internal/cmd/gocdk/deploy.go

```diff
@@ -72,7 +72,7 @@ func deploy(ctx context.Context, pctx *processContext, biome, dockerImage string
 
 	// Run "biome apply".
 	if apply {
-		if err := biomeApply(ctx, pctx, biome, true); err != nil {
+		if err := biomeApply(ctx, pctx, biome, nil); err != nil {
```

Comment:
Note that this changes the value for `Input` from `true` to `false` -- this means that if you run `gocdk deploy` and Terraform has questions for you, it will fail; I think that's OK but let me know what you think.

Our normal workflow in the tool doesn't use Terraform variables, so it's unlikely to happen (the user would have to manually add a variable).

We could also turn this into a flag.

Reply:
This change seems okay, and it would be easy to relax it later by adding a flag, as you say.